agent/component/component.go

```diff
@@ -19,12 +19,13 @@ import (
 type Component interface {
 	// Connect to a NATS Server
 	Connect(bCtx *env.BubblyContext) error
+	Close()
 	// Subscribe to publications on a given subject
 	Subscribe(bCtx *env.BubblyContext, sub DesiredSubscription) (*nats.Subscription, error)
 	// BulkSubscribe to the component's DesiredSubscriptions
 	BulkSubscribe(bCtx *env.BubblyContext) ([]*nats.Subscription, error)
 	// Publish a publication
```

Comment:
With all due respect, this is a comment of the type 

```c
q++; // increment q by one
```

😅

Reply:
Not my comment :)